python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -731,13 +746,13 @@ def _actually_copyFrom(self, _srcobj, _ignore_atts):
             if not self._schema.hasAttribute(name):
                 #raise ValueError('copyFrom: incompatible schema: source=%s destination=%s'%(_getName(_srcobj), _getName(self)))
                 if not hasattr(self, name):
-                    setattr(self, name, self._schema.getDefaultValue(name))
+                    setattr(self, name, self._schema.getDefaultValue(name, make_copy=True))
```

Comment:
If this function is *always* going to be called with `make_copy` explicitly set (for the reasons you say) then why is it an optional argument?